src/main/scala/au/csiro/variantspark/algo/RandomForest.scala

```diff
@@ -220,7 +227,7 @@ class RandomForest(params:RandomForestParams=RandomForestParams()
     val oobAggregator = if (actualParams.oob) Option(new VotingAggregator(nLabels,nSamples)) else None
 
     val builder = modelBuilderFactory(DecisionTreeParams(seed = rng.nextLong, randomizeEquality = actualParams
-      .randomizeEquality))
+      .randomizeEquality, maxDepth = actualParams.maxDepth, minNodeSize = actualParams.minNodeSize))
```

Comment:
Nice to have:
Refactoring: Perhaps it's worth adding a method to `RandomForestParam` to that produces `DecisionTreeParams` and only takes the seed.
E.e. `RandomForestParam.toDecisionTreeParams(seed:Int)`

Reply:
Do you mean something like this?
def toDecisionTreeParams(seed:Int): DecisionTreeParams = { 
       DecisionTreeParams(seed = seed, randomizeEquality = actualParams.randomizeEquality, maxDepth = 
            actualParams.maxDepth, minNodeSize = actualParams.minNodeSize)
}

Such that the above call can be changed to:
val builder = modelBuilderFactory(RandomForestParam.toDecisionTreeParams(seed = rng.nextLong))

Is my understanding correct about this comment?

Reply:
Yes (mostly). This should be an instance method (not a class method)

``` In RandomForestParam
def toDecisionTreeParams(seed:Int): DecisionTreeParams = {
DecisionTreeParams(seed = seed, randomizeEquality = randomizeEquality, maxDepth =
maxDepth, minNodeSize = minNodeSize) // or minNodeSize = this.minNodeSize
}

Usage (no need to pass arguments by name)
```val builder = modelBuilderFactory(actualParams.toDecisionTreeParams(rng.nextLong))```

Reply:
Updated the same